File: modules/wem-api/src/main/java/com/enonic/wem/api/content/data/ContentData.java
package com.enonic.wem.api.content.data;


import com.enonic.wem.api.data.DataSet;
import com.enonic.wem.api.data.RootDataSet;

public final class ContentData
    extends RootDataSet
{
    public ContentData()
    {
        // default
    }

    public ContentData( final RootDataSet rootDataSet )
    {
        super( DataSet.newDataSet( rootDataSet ) );
    }
}

File: modules/wem-webapp/src/main/webapp/admin/common/js/content/ContentData.ts
module api.content {

    export class ContentData extends api.data.RootDataSet {

        constructor() {
            super();
        }

        equals(o: api.Equitable): boolean {

            if (!(o instanceof ContentData)) {
                return false;
            }

            return super.equals(o);
        }
    }
}